src/helpers/email.helper.ts

```diff
@@ -0,0 +1,44 @@
+import * as nodemailer from 'nodemailer';
+import { logger } from '../helpers/logger';
+
+export const sendImageEmailNotification = (
+  emails: string[],
+  displayName: string,
+  imageId: number,
+) => {
+  const getTrans = () => {
+    return nodemailer.createTransport({
+      host: process.env.MAILER_HOST,
+      port: 587,
```

Comment:
Would it make sense if this is coming from the env file too? We could maybe have different setting for development and production environment. This goes for both port and secure properties